grype/presenter/cyclonedxvex/vulnerability_test.go

```diff
@@ -144,3 +145,59 @@ func TestNewVulnerability_AlwaysIncludesSeverity(t *testing.T) {
 		})
 	}
 }
+
+func TestNewVulnerability_AddsFixedVersion(t *testing.T) {
+	tests := []struct {
+		name             string
+		match            match.Match
+		metadataProvider *metadataProvider
+		expected         *[]cyclonedx.Property
+	}{
+		{
+			name: "No known fixed version",
+			match: match.Match{
+				Vulnerability: vulnerability.Vulnerability{
+					Fix: vulnerability.Fix{
+						State:    v3.NotFixedState,
+						Versions: []string{},
+					},
+				},
+				Package: pkg.Package{},
+				Details: nil,
+			},
+			metadataProvider: &metadataProvider{},
+			expected:         &[]cyclonedx.Property{},
+		},
+		{
+			name: "No known fixed version",
```

Comment:
The test name has been repeated here - Did you mean for this to be the case where a fixed version is shown?